executorlib/base/executor.py

```
@@ -0,0 +1,148 @@
+            int: The length of the executor.
+        """
+        return self._future_queue.qsize()
+
+    def __del__(self):
+        """
+        Clean-up the resources associated with the Executor.
+        """
+        try:
+            self.shutdown(wait=False)
+        except (AttributeError, RuntimeError):
+            pass
```

Ruff

Use `contextlib.suppress(AttributeError, RuntimeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(AttributeError, RuntimeError)`

(SIM105)